linux-anvil2/install_packages

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash -il
+
+if [ "$(uname -m)" = "x86_64" ]; then
+   export supkg="gosu"
+   export condapkg="Miniconda3-4.3.21-Linux-x86_64.sh"
+else
+   export supkg="su-exec"
+   export condapkg="Miniconda3-4.3.27-Linux-ppc64le.sh"
+fi
+
+# give sudo permission for conda user to run yum (user creation is postponed
+# to the entrypoint, so we can create a user with the same id as the host)
+echo 'conda ALL=NOPASSWD: /usr/bin/yum' >> /etc/sudoers
+
+# Install the latest Miniconda with Python 3 and update everything.
+curl -s -L https://repo.continuum.io/miniconda/$condapkg > miniconda.sh && \
+    openssl md5 miniconda.sh | grep c51bc5fb00da12b487959b5adf0133ab && \
```

Comment:
This checksum is specific to `$conda_pkg` right?

Reply:
yes..good catch. I'll need to set that above as well